### ISSUE TITLE
Add tests + coverage again

### ISSUE THREAD

**Issue by MichaelDeBoey** [clauderic/react-infinite-calendar#115]
Since `v2.0.0` all the tests + coverage are gone.
It would be nice if they came back, so we at least know that something is broken if we change code for a PR 🙂

CC: @clauderic

**Comment by clauderic** [comment]
Definitely, just need to find time to convert the existing tests. PRs are welcome if you feel like take a stab at it.

**Comment by MichaelDeBoey** [comment]
@clauderic Any news on this one? 🙂
